timing-entrytypes-registry/registry.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8" />
+<title>Registry test for Performance Timeline</title>
+<link rel="help" href="https://w3c.github.io/timing-entrytypes-registry/">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+const STEPS = {};
+
+const types = (self.PerformanceObserver

```

Comment:
Can we also assert_unreached in the async_tests so we early abort if supportedEntryTypes is not defined?

Reply:
this is taken care of by  if (!types || !types.includes(def[0])) return;